docs/libvirt-pod-networking.md

```diff
@@ -0,0 +1,118 @@
+# Using CNI Interfaces and libvirt for networking
+
+Author: Fabian Deutsch \<fabiand@redhat.com\>
+
+## Introduction
+
```

Comment:
How about the following text, I find it clearer:

Today networking in KubeVirt is not integrated with Kubernetes. In order to have network connectivity inside the VM, a user must create a specific VM Spec that induces a particular libvirt domxml, in order to reuse a pre-exising host network interface.

This is obviously suboptimal: